e2e/react/src/react-rsbuild.test.ts

```diff
@@ -0,0 +1,102 @@
+import {
+  checkFilesExist,
+  cleanupProject,
+  newProject,
+  runCLI,
+  runCLIAsync,
+  uniq,
+} from '@nx/e2e/utils';
+
+describe('Build React applications and libraries with Rsbuild', () => {
```

Comment:
Can we add a test that runs Playwight e2e? To be sure that the server actually works and e2e is configured correctly out of the box.

Should we also make the port 4200 to match the other setups?

Reply:
Yep, can update the port and make sure playwright tests are running.